Formula/grokj2k.rb

```diff
@@ -77,23 +73,30 @@ def install
               "\\1 #{Formula["little-cms2"].opt_include}"
     end
 
-    # Workaround to fix build when using Homebrew little-cms2 headers with C++17.
-    # lcms2.h:1279:44: error: ISO C++17 does not allow 'register' storage class specifier
-    # See https://github.com/GrokImageCompression/grok/issues/241
-    ENV.append "CXXFLAGS", "-DCMS_NO_REGISTER_KEYWORD=1"
-
     perl = DevelopmentTools.locate("perl")
+    perl_archlib = Utils.safe_popen_read(perl.to_s, "-MConfig", "-e", "print $Config{archlib}")
+    args = %W[
+      -DGRK_BUILD_DOC=ON
+      -DGRK_BUILD_LIBPNG=OFF
+      -DGRK_BUILD_LIBTIFF=OFF
+      -DPERL_EXECUTABLE=#{perl}
+      -DCMAKE_INSTALL_BINDIR=libexec/bin
+    ]
+    # We install binaries into libexec/bin since we need to create wrapper env scripts
+    # and there is a risk of breakage if we manually move the binaries afterward.
+    # Since pkg-config files are created based on CMAKE_INSTALL_BINDIR, we need to overwrite
+    # these so that users get the path to env script that enables exiftool features.
+    inreplace ["src/lib/core/libgrokj2k.pc.cmake.in", "src/lib/codec/libgrokj2kcodec.pc.cmake.in"],
+              "@CMAKE_INSTALL_BINDIR@", "bin"
```

Comment:
Noting that currently the executables can be manually moved from `bin` to `libexec/bin` without any impact. So, wanted to check if we wanted to use previous installation mention (CMake install to `bin`, manually move to `libexec/bin`) or let CMake directly install to `libexec/bin`.

Reply:
Do you want to do this before merging?  Now we know everything builds at least.